src/utils.ts

```diff
@@ -118,6 +118,37 @@ export const compareText = (actual: string, expected: string, { ignoreCase = fal
     }
 }
 
+export const compareTextWithArray = (actual: string, expectedArray: Array<string>, { ignoreCase = false, trim = false, containing = false}) => {
+    if (typeof actual !== 'string') {
+        return {
+            value: actual,
+            result: false
+        }
+    }
+
+    if (trim) {
+        actual = actual.trim()
+    }
+    if (ignoreCase) {
+        actual = actual.toLowerCase()
+        expectedArray = expectedArray.map(item => item.toLowerCase())
+    }
+    if (containing) {
+        let textInArray = false
```

Comment:
The loop can be removed
```suggestion
        const textInArray = expectedArray.some((t)=> actual.includes(t))
```

Reply:
Done